pkg/cloudprovider/plugins/azure/azure_test.go

```diff
@@ -177,8 +212,19 @@ var _ = Describe("Azure", func() {
 						"credentials": []byte(credential),
 					},
 				}
+				_ = fakeClient.Update(context.Background(), secret)
+
+				c := newAzureCloud(mockAzureServiceHelper)
 
-				_ = fakeClient.Create(context.Background(), secret)
+				err := c.AddProviderAccount(fakeClient, account)
```

Comment:
could you make sure this is not retry-able error

Reply:
Done